src/main/java/com/zemoso/greencommute/repository/JobSkillsRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.zemoso.greencommute.repository;
+
+import com.zemoso.greencommute.entity.JobSkills;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface JobSkillsRepository extends JpaRepository<JobSkills, Integer> {
+
+    @Query(value = "insert into job_skills values (?1, ?2)", nativeQuery = true)
+    void saveByUserIdAndSkillId(int userId, int skillId);
+
```

Comment:
should this be savedJobs ? why saving a job adds entry to job_skills table ?

Reply:
This was fixed in another PR. `feat: network layer`. Had issues with it myself also.

Reply:
Refer to that PR for all the service and network layer latest code.